mpas_analysis/ocean/ohc_timeseries.py

```diff
@@ -36,12 +36,14 @@ def ohc_timeseries(config, streamMap=None, variableMap=None):
 
     # read parameters from config file
     casename = config.get('case', 'casename')
-    ref_casename_v0 = config.get('case', 'ref_casename_v0')
-    indir_v0data = config.get('paths', 'ref_archive_v0_ocndir')
+    ref_casename_v0 = config.get('case', 'ref_casename')
+    indir_v0data = config.get('ocean_reference', 'basedir')
```

Comment:
This seems like a workable solution to me, except that I would want to remove mention of v0 and generalize that to something less ACME specific.

I'm not a huge fan of "casename" for the same reason, but I think it's reasonable for a user running standalone MPAS components to make up a `casename` that's useful to them.  We just need to make sure its purpose is well documented (e.g. determining the image file names and going into the figure titles as a kind of legend).